demos/list.html

```diff
@@ -116,47 +116,47 @@
         <section>
           <h2>Custom Colors</h2>
           <section>
-            <h3>Example - Two-line Lists, Avatars, end detail, inset dividers</h3>
+            <h3>Example - Two-Line Lists, Avatars, Meta Data, Inset Dividers</h3>
```

Comment:
If we're making metadata one word now then we need to update these instances in the demo page.

Reply:
Done